server/odc-core/src/main/java/com/oceanbase/odc/core/flow/graph/Graph.java

```diff
@@ -39,7 +40,9 @@
 @Slf4j
 public class Graph {
 
+    @Getter
```

Comment:
how about moving this package to odc-common? `com.oceanbase.odc.common.graph`

Reply:
sure, I've moved this package